submissions/boladeMarindoti/css/styles2.css

```diff
@@ -0,0 +1,240 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
please rename this file to `product.css` instead

Reply:
I have  renamed the file to product.css